### ISSUE TITLE
feat: POS RAN Model (Positioning for AI-RAN)

### ISSUE THREAD

**Issue by thc1006** [thc1006/new_frontendv3#18]
## 概述
在 AI-RAN Applications 頁面中實作 POS RAN Model (Positioning for AI-RAN) 功能。

## Figma 參考
- [Model Choose (277:1146)](https://www.figma.com/design/P7bPVphHwCiPDTQGRVcAK3?node-id=277-1146)
- [Finetuning Running (277:993)](https://www.figma.com/design/P7bPVphHwCiPDTQGRVcAK3?node-id=277-993)

## 與 AI Simulator 的 NES/Positioning 差異

| 項目 | AI Simulator NES | POS RAN |
|------|-----------------|---------|
| 主要按鈕 | Pre-train, Preview | Finetune, Enable, Re-train |
| Loss Function | Self-defined | MSE |
| 圖表數量 | 3 個（Reward, Critic, Actor） | 1 個（Training + Validation） |
| 圖表線條 | 單線 | 雙線（training/validation） |

### 操作按鈕（左側面板）
| 按鈕 | 顏色 | 狀態 | 功能 |
|------|------|------|------|
| Finetune | 綠色 | 啟用 | 微調模型 |
| Enable | 灰色 | 禁用/啟用 | 啟用模型 |
| Re-train | 灰色 | 禁用/啟用 | 重新訓練 |

### 底部按鈕
- BACK（藍色）
- STOP（紅色，訓練中顯示）

### Training Process 面板（右側）
- [ ] 標題：Training Process
- [ ] Training Status: running（紅色）/ Finish（綠色）
- [ ] Loss Function: **MSE**
- [ ] 單一圖表：Loss over Epoch
  - 藍色實線：training
  - 橙色虛線：validation
- [ ] 訓練資訊：
  - Epoch: 500/500
  - Training Loss: 0.09
  - Validation Loss: 0.09

## 技術實作
- 新增 `PosRanModel` 元件
- 新增 `MseLossChart` 元件（雙線圖）
- 複用 `TrainingStatusCard` 元件
- 狀態管理：idle → finetuning → finish

## 後端 API (需 placeholder)
```
POST   /projects/{id}/ai-ran/positioning/finetune/start
POST   /projects/{id}/ai-ran/positioning/finetune/stop
GET    /projects/{id}/ai-ran/positioning/finetune/status
POST   /projects/{id}/ai-ran/positioning/enable
POST   /projects/{id}/ai-ran/positioning/retrain
```

## 相關 Issues
- #17 AI-RAN Applications 頁面

## 驗收條件
- [ ] 頁面 UI 對齊 Figma 設計
- [ ] Finetune 流程狀態切換正常
- [ ] MSE Loss 圖表顯示雙線（training/validation）
- [ ] Training Status 正確顯示
- [ ] E2E 測試覆蓋
